src/components/studies/scheduler/ScheduleCreatorTab.tsx

```diff
@@ -49,12 +50,12 @@ const useStyles = makeStyles((theme: Theme) =>
       fontStyle: 'normal',
       fontWeight: 600,
     },
-    assessments: {
+    readOnlyAssessmentContainer: {
```

Comment:
this is the same as assessments. Let's reuse this class and add the bgColor update as a style